### PR TITLE
chore: Alert Slack when CI fails on main

### PR DESCRIPTION
**Pull request by Callisto13**
I am not sure if the `if` syntax is correct, running some tests now but if anyone sees an obvious mistake, plz lmk.

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -291,6 +291,26 @@ jobs:
...
 
+  notify-failure:
+    name: Send Slack alert
+    needs:
+      - ci-go
+      - ci-static
+      - ci-js
+      - ci-build-gitops-image
+      - ci-upload-images
+      - ci-upload-binary
```
Comment (ozamosi) [inline .github/workflows/pr.yaml:302]: I think you can replace this list with just `finish-ci-merge`

**Review by ozamosi — APPROVED** [review]
If syntax _looks_ correct

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (Callisto13) [reply .github/workflows/pr.yaml:302]: i did start with that, changed for... reasons i forget

**Comment by Callisto13** [comment]
tested it works, and made changes. @ozamosi you want to re-review or all good?

**Comment by ozamosi** [comment]
All good!

**Pull request merged**